.github/workflows/build-swift.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Build Swift Package
+
+on:
+  push:
+    branches:
+      - "rc"
+      - "hotfix-rc"
+      - "master"
+      - "DEVOPS-1525-swift-workflow-for-package-publishing"
```

Comment:
```suggestion
```

Was this for testing?

Reply:
Yep, I can remove it for now, but I may add it back if there are some changes.